### PR TITLE
Poppy Reporting Update

### PR DESCRIPTION
**Pull request by adoerrer**
This updates the Reporting with LDP/Metadb section for the Poppy release.

### CODE REVIEW

File: content/en/docs/Reporting/FOLIO Analytics/_index.md [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
 linkTitle: "FOLIO Analytics"
-date: 2021-06-24
-weight: 30
+date: 2023-10-25
+weight: 50
 tags: ["parenttopic"]
 ---
 
+{{% alert color="info"%}}
+<b>Note:</b> This documentation covers FOLIO Analytics v1.7.0, which was released at  approximately the same time as the FOLIO Poppy release.
```
Comment (julianladisch) [inline content/en/docs/Reporting/FOLIO Analytics/_index.md]: `<b>Note:</b>` should be replaced with `**Note:**`.
Please don't use HTML that some documentation writers don't understand, use Markdown instead.
Using `<b>` for emphasis is not accessible:
* https://www.w3.org/WAI/WCAG22/Techniques/html/H49.html
* https://www.w3.org/WAI/WCAG22/Techniques/failures/F2.html

File: content/en/docs/Reporting/FOLIO Analytics/_index.md [hunk 2]
```diff
@@ -54,26 +57,27 @@ The following section demonstrates this workflow using [DBeaver](https://dbeaver
...
-        * SSL mode (will likely be “require”)
-    * Note that a hosted LDP reporting database is currently available for the FOLIO community.  It provides access to data from the FOLIO reference environment folio-snapshot and is updated hourly. For login info, please review the [Reporting SIG wiki documentation](https://wiki.folio.org/display/RPT/FOLIO+Reporting+Reference+Environment).
+   * You will need to get the following information from your local LDP/Metadb administrator:
+  	* Host (typically looks like a URL, like ldp.institution.edu)
+  	* Port (typically 5432)
+  	* Database name
+  	* User name and password
+  	* SSL mode (will likely be “require”)
+{{% alert color="info"%}}
+<b> Note </b>that a hosted LDP/Metadb reporting database is currently available for the FOLIO community.  It provides access to data from the FOLIO reference environment folio-snapshot and is updated hourly. For login info, please review the [Library Data Platform testbed documentation](https://librarydataplatform.org/testbed/).{{% /alert %}}
```
Comment (julianladisch) [inline content/en/docs/Reporting/FOLIO Analytics/_index.md]: `<b>Note:</b>` should be replaced with `**Note:**`.
Please don't use HTML that some documentation writers don't understand, use Markdown instead.
Using `<b>` for emphasis is not accessible:
* https://www.w3.org/WAI/WCAG22/Techniques/html/H49.html
* https://www.w3.org/WAI/WCAG22/Techniques/failures/F2.html

File: content/en/docs/Reporting/FOLIO Analytics/_index.md [hunk 3]
```diff
@@ -88,29 +92,32 @@ The following section demonstrates this workflow using [DBeaver](https://dbeaver
 #### Run the SQL query
 
 1. To run the query, either click on the **Execute SQL Script** button on the left side of the script editor (it should be the third button from the top and look like a document with a “play” symbol inside of it) or select **Execute SQL Script** from the **SQL Editor** menu.
-1. The results will hopefully then appear in the **results panel** below the script. Note: When querying parts of the database with a lot of data, like the inventory tables, there may be a long delay before results are returned.
+1. The results will hopefully then appear in the **results panel** below the script. 
+{{% alert color="info"%}}
+<b> Note </b>When querying parts of the database with a lot of data, like the inventory tables, there may be a long delay before results are returned.
```
Comment (julianladisch) [inline content/en/docs/Reporting/FOLIO Analytics/_index.md]: `<b>Note:</b>` should be replaced with `**Note:**`.
Please don't use HTML that some documentation writers don't understand, use Markdown instead.
Using `<b>` for emphasis is not accessible:
* https://www.w3.org/WAI/WCAG22/Techniques/html/H49.html
* https://www.w3.org/WAI/WCAG22/Techniques/failures/F2.html

File: content/en/docs/Reporting/LDP and Metadb/_index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+title: "LDP and Metadb"
+linkTitle: "LDP and Metadb"
+date: 2023-10-25
+weight: 30
+tags: ["parenttopic"]
+---
+
+
+{{% alert color="info"%}}
+<b>Note:</b> This [WOLFcon 2021 presentation](https://www.youtube.com/watch?v=SM1vq0zvxsY) gives an overview of the Library Data Platform.
```
Comment (julianladisch) [inline content/en/docs/Reporting/LDP and Metadb/_index.md]: `<b>Note:</b>` should be replaced with `**Note:**`.
Please don't use HTML that some documentation writers don't understand, use Markdown instead.
Using `<b>` for emphasis is not accessible:
* https://www.w3.org/WAI/WCAG22/Techniques/html/H49.html
* https://www.w3.org/WAI/WCAG22/Techniques/failures/F2.html

File: content/en/docs/Reporting/LDP and Metadb/_index.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+### Documentation 
+
+For detailed information about how to set up and configure the LDP software, please review this [link](https://github.com/library-data-platform/ldp#readme) to access individual links to guides in the LDP repository. There you also will find the latest versions and fixes of the LDP software.
+
+## Setting up derived tables
+
+To let report users take full advantage of the community-developed [report queries](../folio-analytics/#using-queries-from-the-folio-analytics-repository) as well as having a good starting point for [ad hoc querying](../folio-analytics/#ad-hoc-querying-using-ldp-tables), it is strongly recommended to set up a nightly update on [derived tables](https://github.com/folio-org/folio-analytics/blob/main/sql/derived_tables/README.md  ). 
+
+{{% alert color="info"%}}
+<b>Note:</b>use of *views* and *materialized views* is not supported in LDP databases and may cause the LDP software to be unable to perform data updates.
```
Comment (julianladisch) [inline content/en/docs/Reporting/LDP and Metadb/_index.md]: `<b>Note:</b>` should be replaced with `**Note:**`.
Please don't use HTML that some documentation writers don't understand, use Markdown instead.
Using `<b>` for emphasis is not accessible:
* https://www.w3.org/WAI/WCAG22/Techniques/html/H49.html
* https://www.w3.org/WAI/WCAG22/Techniques/failures/F2.html

Comment (julianladisch) [inline content/en/docs/Reporting/LDP and Metadb/_index.md]: Please add a space before "use".

File: content/en/docs/Reporting/LDP and Metadb/_index.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+### Moving and loading data
+Loading and moving data into an LDP reporting database is as simple as it is for databases in general. 
+
+For Postgres there are two common approaches:
+
+* [COPY](https://www.postgresql.org/docs/current/sql-copy.html): SQL command for moving table data via csv files
+* [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) / [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html): Postgres command line tools for importing and exporting data
+
+## Installing and configuring the Metadb software 
+\* The next generation software platform Metadb has been released and has been adopted by some institutions. Metadb is a new streaming data integration platform intended to eventually replace LDP. For installing and configuring Metadb follow the *Metadb Documentation* at the [Library Data Platform - resources site](https://librarydataplatform.org/resources/).
```
Comment (julianladisch) [inline content/en/docs/Reporting/LDP and Metadb/_index.md]: Can the `\*` been removed?

File: content/en/docs/Reporting/FOLIO Analytics/_index.md (hunk 1 above)
Comment (adoerrer) [reply content/en/docs/Reporting/FOLIO Analytics/_index.md]: Thanks

File: content/en/docs/Reporting/FOLIO Analytics/_index.md (hunk 3 above)
Comment (adoerrer) [reply content/en/docs/Reporting/FOLIO Analytics/_index.md]: Thanks, done.

File: content/en/docs/Reporting/LDP and Metadb/_index.md (hunk 6 above)
Comment (adoerrer) [reply content/en/docs/Reporting/LDP and Metadb/_index.md]: Again thanks for the catch

**Pull request merged**